schemas/content/product.schema.json

```diff
@@ -60,81 +53,68 @@
           "description":
             "The internal unique ID of the product in the commerce backend system."
         },
-        "xdm:masterProductSKU": {
-          "title": "Master Product SKU",
-          "type": "string",
+        "xdm:prices": {
+          "title": "Prices",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "properties": {
+              "currency": {
+                "title": "Currency Code",
+                "type": "string",
+                "examples": ["USD", "EUR"],
+                "pattern": "^[A-Z]{3}$",
+                "description":
+                  "The ISO 4217 alphabetic currency code used for cost and pricing."
+              },
+              "amount": {
+                "title": "Amount",
+                "type": "number",
+                "minimum": 0,
+                "description":
+                  "The amount value for that price."
```

Comment:
strange wording to my eye.

Reply:
You mean `amount`, it is widely used for price or cost objects in commerce. A price always has currency and value/amount/number value.